zulipterminal/themes/solarized_dark.py

```diff
@@ -0,0 +1,90 @@
+"""
+SOLARIZED DARK
+--------------
+
+
+"""
+
+from pygments.styles.solarized import SolarizedDarkStyle
+
+from zulipterminal.themes.colors_solarized import DefaultBoldColor as Color
+
+STYLES = {
+    # style_name       :  foreground                   background
+    None               : (Color.base0,                 Color.base03),
```

Comment:
While the dark and light themes use some similar colors from the solarized palette, `base03` is not very descriptive when we're writing the styles. My suggestion would be to write two separate enums for dark (and later light) versions which use the values from the main palette. We could still store the main colors in an enum if that's simpler.

This would allow eg. `Color.bg`, `Color.fg` (or longer, like background/foreground) to be different values based on which enum we import, which is clearer than `base0`.

If we keep the highlight colors the same, this could potentially also enable complete reuse of `STYLES` for light after importing a different enum for `Color`.

Reply:
A simple approach could be to have the core palette as you have it now - which is easier to compare to the solarized specification - and then define other enums for dark and light which reuse those values, eg.

```python3
class SolarizedDarkColor(Enum):
    background = SolarizedColor.base03.value
    foreground = SolarizedColor.base0.value
    ...
```
Then in the theme file we can import that enum instead, and `Color.foreground` is easier to remember and validate than some number. Similarly the enum as above makes it easier to validate that base0 is the foreground and should be according to the specification.

Please do read the enum documentation yourself, but I believe inheritance of enums with values doesn't work like 'sets' in this way, ie. I don't think you can use a 'mixin enum' to set common colors.

Reply:
ok got it, will try this out and update soon